ios/BetterRailWidget/Live Activity/Views/CountdownView.swift

```diff
@@ -9,12 +9,10 @@ struct CountdownView: View {
   }
   
   var color: Color {
-    if (delay >= 5) {
-      return .red
```

Comment:
I think we should keep it, the user need to be alerted that he may miss his connection.